crates/terminal/src/lib.rs

```diff
@@ -129,6 +129,24 @@ macro_rules! ceprint {
     };
 }
 
+pub fn print_link(url: &str) {
```

Comment:
Would it be cleaner if this didn't directly do the printing and just returned a `Cow<str>` of the escaped string so that it can be used at the callsite by a `println!` macro?